modules/local/gffread.nf

```diff
@@ -0,0 +1,51 @@
+process GFFREAD {
+    tag "$gff"
+    label 'process_low'
+
+    conda "bioconda::gffread=0.12.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gffread:0.12.1--h8b12597_0' :
+        'biocontainers/gffread:0.12.1--h8b12597_0' }"
+
+    input:
+    tuple val(meta), path(gff)
+    tuple val(meta2), path(fasta), path(fai)
+
+    output:
+    path "*${prefix}.gtf"        , emit: gtf,        optional: true
+    path "*${prefix}.fa"         , emit: tr_fasta,   optional: true
```

Comment:
Do we want to output it with the meta as well? we can also leave it for a later implementation if deemed necessary.

Reply:
I will add it in one of the next PR, I want to add documentation and clarification comments following best practices, I will also change all the local module to have input/output with meta when I do so